src/Template/Main/claims.ctp

```diff
@@ -101,6 +106,17 @@ if (strlen(trim($desc)) == 0) {
 
             <div class="clear"></div>
             <?php endif; ?>
+            
+            <div class="half-width"><div class="label help-text" title="The quantity of tokens used to back the stake.">Claim bid</div></div>
+            <div class="half-width"><div class="label help-text" title="The sum of the claim bid and all of its active supports.">Effective amount</div></div>
+
+            <div class="value half-width"><?php echo (float) ($claim->claim_bid) . ' LBC'; ?></div>
+            <div class="value half-width"><?php echo ($effective_amount < $claim->claim_bid) ? '<em>Synchronizing</em>' : ((float) ($claim->effective_amount / 100000000) . ' LBC');  ?></div>
```

Comment:
same thing here..using the const would be best...why not just fix the format currency function?

Reply:
I added a function in AmountHelper to format LBRY prices, to make it more reuseable.